example.py

```diff
@@ -1,20 +1,18 @@
 import gym_electric_motor as gem
-from gym_electric_motor import reference_generators as rg
 import gem_controllers as gc
 from gym_electric_motor.visualization import MotorDashboard
 from gym_electric_motor.physical_system_wrappers import FluxObserver
 
 
-env_id = 'Cont-CC-SCIM-v0'
+env_id = 'Cont-TC-SCIM-v0'
 
 # Initialize the motor environment. The controller of the induction motor requires a flux observer.
 env = gem.make(
         env_id,
         physical_system_wrappers=(FluxObserver(),),
         visualization=MotorDashboard(state_plots=['omega', 'torque', 'i_sd', 'i_sq', 'u_sd', 'u_sq', 'psi_angle', 'psi_abs']),
-    )
```

Comment:
The example has now been fitted to only work with the SCIM. Either name it that way and also provide an example for another motor or catch it via some case distinction such that one example file can be used for all motor classes.